### ISSUE TITLE
Error: Invalid JSON data

### ISSUE THREAD

**Issue by jlliao** [hyperledger/composer#4138]
Unhandled error for request POST /api/Prosumer: Error: Invalid JSON data. Found a value that is not a string: 100 for relationship RelationshipDeclaration {name=coins, type=org.example.mynetwork.Coins, array=false, optional=false}

What is the problem here?

**Comment by kOstovic** [comment]
your syntax for calling relationship was probably wrong,this should do it

"RelationshipDeclaration": "resource:org.example.mynetwork.Coins#100",

**Comment by LeoMoonStar** [comment]
Yep, @kOstovic is right. It looks like we need pass a json type inside the bracket, but actually what we need to do is delete the bracket and pass the type#id.
I think REST api may need to delete the bracket, it misleads
